chain/chain/src/tests/simple_chain.rs

```diff
@@ -62,17 +63,17 @@ fn build_chain() {
     assert_eq!(chain.head().unwrap().height, 4);
     let count_instant = Clock::instant_call_count();
     let count_utc = Clock::utc_call_count();
-    assert_eq!(count_utc, 5);
+    assert_eq!(count_utc, 9);
     assert_eq!(count_instant, 0);
     #[cfg(feature = "nightly_protocol")]
     assert_eq!(
         chain.head().unwrap().last_block_hash,
-        CryptoHash::from_str("BNap11nsM7PEqYQertYU487g7gkCteQ8Fmee6QfyRdVQ").unwrap()
+        CryptoHash::from_str("JDNJcU8dbsY18Trzbts9gSphtQqw9C5fx1Abg3SzDdRZ").unwrap()
```

Comment:
can you explain why the header has changed?

Reply:
I will revert them.